<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WordReview</class>
 <widget class="QDialog" name="WordReview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>253</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="top_label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>66</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>211</width>
     <height>66</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="word_label">
        <property name="text">
         <string>单词：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="word_lineedit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="meaning_label">
        <property name="text">
         <string>释义：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="meaning_lineedit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>120</y>
     <width>82</width>
     <height>68</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPushButton" name="sound_button">
      <property name="text">
       <string>发音</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="new_button">
      <property name="text">
       <string>生词</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="close_button">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>200</y>
     <width>87</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QPushButton" name="next_button">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>200</y>
     <width>87</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>下一个</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>60</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>all</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
